test-base.cfg

```diff
@@ -215,7 +221,7 @@ initialization =
         shell=True, stdout=subprocess.PIPE).stdout.read().strip()
     if '--absolute' in sys.argv: pkgdir = os.path.abspath(pkgdir) ; sys.argv.remove('--absolute')
     if len(sys.argv) < 2: sys.argv.extend([pkgdir])
-    sys.argv.extend(${pycodestyle:args})
+    sys.argv.extend(${pycodestyle-ci-run:args})
```

Comment:
There's something odd now: the `initialization` option is now in the `[pycodestyle-ci-run]` section, but this won't be used by the `collective.recipe.template`.

I think `bin/pycodestyle` does no longer include the intialization code and therefore does no longer do the same thing as before.

`bin/pycodestyle` should work the same as before (it should limit to the source directory of this package).